python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4279,8 +4267,8 @@ def test_df_astype_to_categorical_ordered(ordered):
     gdf = DataFrame.from_pandas(pdf)
 
     assert_eq(
-        gdf.astype("int32", ordered=ordered),
-        gdf.astype("int32", ordered=ordered),
```

Comment:
Same as above, although I think I myself added this test so insert "spiderman pointing at spiderman" meme here.

Reply:
https://i.kym-cdn.com/entries/icons/mobile/000/023/397/C-658VsXoAo3ovC.jpg